src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -207,6 +207,13 @@ elementFormDefault="qualified">
                   </xs:documentation>
                 </xs:annotation>
               </xs:element>
+              <xs:element name="GeneratePathProperty">
+                <xs:annotation>
+                  <xs:documentation>
+                    <!-- _locID_text="PackageReference_GeneratePathProperty" _locComment="" -->Set to true to generate a Pkg* property that points to the restored location of the NuGet package contents
```

Comment:
default?

Reply:
I was hoping someone more knowledgeable than me could push directly to this branch ;) 

Did you just fall into my trap?

Reply:
```suggestion
                    <!-- _locID_text="PackageReference_GeneratePathProperty" _locComment="" -->Set to true to generate a Pkg* property that points to the restored location of the NuGet package contents. If unset, does not create the property.
```

Reply:
Maybe, but I do one-line changes from the browser to maximally ping you in the process 🙂